components/js/common.js

```diff
@@ -0,0 +1,6 @@
+import { GoogleAnalytics } from './google/analytics';
+
+export function CommonJS() {
+    const jsx = <GoogleAnalytics trackingId="G-6HNBH1SW7K" />;
```

Comment:
Use this as the tracking id instead: `UA-43807579-1`

Reply:
done